src/modules/common/components/social-links-editor.jsx

```diff
@@ -78,7 +78,7 @@ export default class SocialLinksEditor extends React.Component {
       .filter(item => socialUrls.map(url => url.site).indexOf(item) < 0);
     return (
       <div>
-        <table className="edit-social-links">
+        <table className="edit-social-links form__input--top-margin">
```

Comment:
I really don't think a class intended for form defaults should be reused on tables even if it's just for applying a margin. This sets us up for unintended collisions which is an issues with PFE's current CSS.